### ISSUE TITLE
Irreversible airflow obstruction

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6550]
**Preferred term label:** Irreversible airflow obstruction

**Synonyms**
Irreversible pulmonary obstruction

**Definition (free text, please give PubMed ID)**
Airflow obstruction without significant response to a bronchodilator defined as an increase in  FEV1 and/or FVC by 12 percent or more and by at least 200 mL. PMID: 16264058

**Parent term (use hpo.jax.org/app)**
Pulmonary obstruction HP:0006536

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
Primary label airflow obstruction, syn, Pulmonary obstruction

**Comment by pnrobinson** [comment]
added
